lib/systems/examples.nix

```diff
@@ -63,6 +63,20 @@ rec {
     platform = platforms.fuloong2f_n32;
   };
 
+  muslpi = raspberryPi // {
+    config = "armv6l-unknown-linux-musleabihf";
+    libc = "musl";
```

Comment:
Are the `libc = ...`s needed? I'd love to remove them everywhere and have inference take care of it; the more minimal these attribute sets are the better!

Reply:
I don't think so! This one is "needed" only because I base it off of "raspberryPi" which explicitly sets libc to glibc; if it didn't I wouldn't need to override it.

AFAIK they're inferred properly by default in all examples in this file.

Which, by the way, is magical :D.